lib/scientist/experiment.rb

```
@@ -282,4 +267,9 @@ def raise_on_mismatches?
       !!raise_on_mismatches
     end
   end
+
+  private
+  def missing_behaviour(behaviour)
```

Please inline this `raise` instead of extracting a `missing_behavior` method.